google-oauth-client/src/main/java/com/google/api/client/auth/oauth2/JWTAuthentication.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright (c) 2020 Google Inc.
```

Comment:
Incorrect copyright format. Never use parentheses for this

Reply:
Happy to make the change. It's literally copied straight from ClientParameterAuthentication though.

Reply:
Pushed requested change.